launch/client.py

```diff
@@ -832,6 +835,7 @@ def batch_async_request(
         self,
         bundle_name: str,
         urls: List[str],
+        inputs: Optional[List[Dict[str, Any]]],
```

Comment:
should we do `urls: Optional[List[str]] = None` and `inputs: Optional[...] = None`?

Reply:
omg yea my b